tools/chplvis/Makefile

```diff
@@ -56,7 +53,9 @@ OFILES= $(SRCS:.cxx=.o) $(GENSRCS:.cxx=.o)
 all: chplvis
 
 chplvis: $(GENSRCS) $(OFILES) 
-	$(CXX) -o chplvis $(OFILES) $(LDFLAGS) $(LIBS)
+	$$($(FLTK_CONFIG) --cxx)  -o chplvis $(OFILES) \
+	$$($(FLTK_CONFIG) --ldflags) $$($(FLTK_CONFIG) --libs)
```

Comment:
Think it might be helpful to indent this second line a few spaces, to make it more obvious that it's a continuation of the command starting on the previous line, rather than a new command. (The trailing backslash on the previous line does say this, but someone reading rapidly might not pick that up.)